example-script-runtime-service/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/alfresco/alfresco-script-app-runtime:develop
+
+COPY project/scripts /maven/scripts
+COPY project/project.json /maven/project.json
```

Comment:
"COPY project/project.json /maven/project.json" is one of the changes with the upgrade in m6, we won't need this file. We can remove it from here, and i need to remove it later from the other projects except process-runtiime

Reply:
ok I just copied from DMN, can remove this